.github/workflows/helm-chart-tests.yaml

```
@@ -0,0 +1,834 @@
+          service-name: ${{ matrix.plans.service-name }}
+          environment: dev
+          environment-type: dev
+          release-description: helm-chart-${{ needs.package-and-push-helm-chart.outputs.chart-version }}
+          spec-type: ServicePlanSpec
+
+  omnistrate-e2e-tests:
+    name: Omnistrate E2E Tests
+    runs-on: ubuntu-latest
+    needs: update-omnistrate-plans
+    # if: ${{ needs.update-omnistrate-plans.result == 'success' }}
+    if: false
```

actionlint (1.7.9)

condition "false" is always evaluated to false. remove the if: section

(if-cond)